website/www/site/static/images/logos/sdks/java-gray.svg

```diff
@@ -0,0 +1,28 @@
+</defs>
+</svg>
```

Comment:
Where did the Java/Python logos come from? I know some language icons have special licenses we need to respect.

Reply:
Honestly I don't know, but I will ask our UI team about this.

Reply:
Icons where changed, it should be good now

Reply:
Thanks!